src/constants/validation.constants.ts

```diff
@@ -174,11 +174,11 @@ export const patterns = {
      */
     number: /\d/,
     /**
-     * @regex /\W/
-     * @description This pattern matches any string that contains special characters.
+     * @regex /[^\w\s£]/
```

Comment:
What we are customising this?
Utils is a place for having generic functions I believe it's not a good idea to exclude British from here 🤔
cc: @amam-deriv @yashim-deriv